tools/kmodules/lisa-in-tree/fetch_lisa_module.py

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env python3
+
+import os
+import shutil
+import argparse
+import subprocess
+
+from pathlib import Path
+
+
+def main():
+    parser = argparse.ArgumentParser(description="""
+    Fetch the Lisa module from GitHub or a local copy and integrate it
+    inside the kernel tree.
+
+    Method 1 - using LISA_HOME.
+
+    If the script finds the LISA_HOME environment variable (present after sourcing the Lisa shell)
+    it will symlink the Lisa module sources from the local Lisa git checkout into the kernel tree.
+    That way updating the local Lisa tree will automatically update the module sources in the kernel
+    tree and the changes will be present after the next kernel rebuild.
+    In this case, the script only needs to be run once.
+
+    Method 2 - Sparse checkout.
+
+    If the script doesn't find LISA_HOME (e.g. building on an external server without a Lisa checkout)
+    the alternative mode is to directly clone the module sources into the tree.
+                                     """)
+    parser.add_argument('-g', '--git-ref', help='Lisa git reference to checkout')
+    parser.add_argument('-f', '--force', action='store_true', help='Override the module checkout if it already exists')
+    parser.add_argument('--module-kernel-path',
+                        help='Path relative to the kernel tree root where the module  will be stored',
+                        default='./private/gs-google/drivers/soc/arm/vh/kernel/lisa')
```

Comment:
I don't think we should provide a default value for something like that. 6 months from now that path will change to whatever Google decides is the new flavor and we will have a broken default forever in that script, with people relying on that and others scratching their head why they don't get the module build because they used the now-wrong default value.